test/spec_session_cookie.rb

```diff
@@ -406,6 +423,11 @@ def call(env)
     response.body.must_equal '"foo"'
   end
 
+  it "Sets same_site option in cookie if session is " do
```

Comment:
Not sure this spec actually tests the same_site option.

Reply:
Ah you're right, I left that spec in by accident after I moved stuff around to what I thought was a more sensible place in the file nearer to other more related functionality. I removed it now. Thanks for looking!